compiler/src/dmd/cond.d

```diff
@@ -691,6 +691,9 @@ extern (C++) final class VersionCondition : DVCondition
             case "LDC":
             case "linux":
             case "LittleEndian":
+            case "LoongArch64":
```

Comment:
perhaps also add `LoongArch32` in this PR, to reserve the version identifier for future use in druntime.

Reply:
Done.